packages/components/src/FormField/FormField.tsx

```diff
@@ -186,20 +186,19 @@ export const FormField = React.forwardRef(
       handleValidation();
     }, [value]);
 
-    const hasErrors = hasErrorMessages(validations);
-
     const wrapperClassNames = classnames(
       styles.wrapper,
       inline && styles.inline,
       size && styles[size],
       align && styles[align],
       errorMessage && styles.hasErrorMessage,
-      (invalid || errorMessage || hasErrors) && styles.invalid,
       disabled && styles.disabled,
       maxLength && styles.maxLength,
       {
         [styles.miniLabel]:
           (hasMiniLabel || type === "time" || type === "select") && placeholder,
+        [styles.invalid]:
+          invalid || errorMessage || hasErrorMessages(validations),
```

Comment:
I've no problem with this diff but curious is this initiated from a new linting rule?